components/properties/src/props.rs

```diff
@@ -624,3 +626,75 @@ impl SentenceBreak {
     pub const LF: SentenceBreak = SentenceBreak(13); // name="LF"
     pub const SContinue: SentenceBreak = SentenceBreak(14); // name="SC"
 }
+
+/// Enumerated property Canonical_Combining_Class.
+/// See UAX #15:
+/// <https://www.unicode.org/reports/tr15/>.
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[repr(transparent)]
+pub struct CanonicalCombiningClass(pub u8);
```

Comment:
Question (not-blocking): is it correct to assume that Canonical_Combining_Class may have new property values defined in the future?

I ask because this is a newtype struct like the other newtypes representing enumerated properties, with a `pub u8` argument. The numerical values (discriminants) of the enums also seem to suggest that space is being left open for new property values, too. Just checking.

Reply:
Yes, it is unusual but possible for new Canonical_Combining_Class values to be added in the future.

Numeric values will always be in the range 0..254.

Reply:
Since it's not UB to materialize a `CanonicalCombiningClass` that wraps a `u8` value for which a named constant doesn't exist (yet), it's not a problem for the value space to grow.